src/components/Upload.tsx

```diff
@@ -47,13 +51,15 @@ function Upload() {
 
   return (
     <div className="flex flex-col items-center gap-8" data-config-loading={isConfigLoading}>
-      <ConfigCard
-        title="Configuration"
-        options={configOptions}
-        value={selectedConfig}
-        isLoading={isConfigLoading}
-        onChange={(value) => setSelectedConfig(value)}
-      />
+      {disableConfig.toLowerCase() === 'true' ? (
```

Comment:
shouldn't `disableConfig` be boolean at this stage in the app? At least that would make more sense to me